doc/sphinx/magnetostatics.rst

```diff
@@ -171,11 +171,11 @@ calculations. The feature ``SCAFACOS_DIPOLES`` has to be added to
 :file:`myconfig.hpp` to activate this feature. At the time of this writing (Feb
```

Comment:
This should probably go: "At the time of this writing (Feb 2018)"

Reply:
done